assets/src/edit-story/elements/image/editCropMovable.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Movable from '../../components/movable';
+import { getFocalFromOffset } from './util';
+
+function EditCropMovable( {
+	setProperties, cropBox, croppedImage,
+	x, y,
+	offsetX, offsetY, imgWidth, imgHeight,
+} ) {
+	const moveableRef = useRef();
+	const cropRef = useRef( [ 0, 0, 0, 0 ] );
+
+	// Refresh moveables to ensure that the selection rect is always correct.
+	useEffect( () => {
+		if ( moveableRef.current ) {
+			moveableRef.current.updateRect();
+		}
+	} );
+
+	return (
+		<Movable
+			ref={ moveableRef }
+			className="crop-movable"
+			targets={ cropBox }
+
+			origin={ false }
+			resizable={ true }
+			onResize={ ( { width: resizeWidth, height: resizeHeight, delta, drag } ) => {
+				const [ tx, ty ] = [ drag.beforeTranslate[ 0 ], drag.beforeTranslate[ 1 ] ];
+				cropBox.style.transform = `translate(${ tx }px, ${ ty }px)`;
+				croppedImage.style.transform = `translate(${ -tx }px, ${ -ty }px)`;
+				if ( delta[ 0 ] ) {
+					cropBox.style.width = `${ resizeWidth }px`;
+				}
+				if ( delta[ 1 ] ) {
+					cropBox.style.height = `${ resizeHeight }px`;
+				}
+				cropRef.current = [ tx, ty, resizeWidth, resizeHeight ];
+			} }
+			onResizeEnd={ () => {
+				cropBox.style.transform = '';
+				croppedImage.style.transform = '';
+				cropBox.style.width = '';
+				cropBox.style.height = '';
+				const [ tx, ty, resizeWidth, resizeHeight ] = cropRef.current;
+				cropRef.current = [ 0, 0, 0, 0 ];
+				if ( resizeWidth === 0 || resizeHeight === 0 ) {
```

Comment:
Could we instead limit the resizing within `onResize`, to ensure it doesn't get to 0? I suppose this should be done within the display mode `Moveable`, too. 

We can follow up on this within a separate PR though.

Reply:
This serves a bit simpler purpose. I noticed that if I just click and release on the anchor, I get a sequence of events: `onResizeStart` and `onResizeEnd`. There's no `onResize` in between. The meaning of this simply that "no resize happened". Thus I simply ignore this sequence here. We had the same problem with the selection transformer and I fixed it elsewhere.

Reply:
Ah, now I understand, makes sense.